pravega/src/test/scala/akka/stream/alpakka/pravega/package.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) since 2016 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.stream.alpakka
+
+import io.pravega.client.stream.Serializer
+import java.nio.ByteBuffer
+import io.pravega.client.stream.impl.UTF8StringSerializer
+
+package object pravega {
```

Comment:
Wouldn't it be more obvious how the serializers get in scope if this was in `package docs.scaladsl` and the a regular `object Serializers`?
By importing from it, the serializers in `PravegaReadWriteDocs` and `PravegaSettingsSpec` could be removed, as well.